tests/test_date_parser.py

```diff
@@ -235,13 +236,13 @@ def test_preferably_past_dates(self, date_string, expected):
         self.given_date_string(date_string)
         self.when_date_is_parsed()
         self.then_date_was_parsed_by_date_parser()
-        self.then_period_is('day')
```

Comment:
Period will be `month` for March [here](https://github.com/scrapinghub/dateparser/pull/59/files#diff-36128d81467a740315c9e5f49860c291L224)